LIMS2DB/objectsDB/objectsDB.py

```diff
@@ -459,8 +459,7 @@ def _make_sample_run_id(self, seq_art, lims_run, prep, run_type):
                 samp_run_met_id = '_'.join([lane, date, fcid, barcode])
             except TypeError: 
                 #happens if the History object is missing fields, barcode might be None
-                logging.debug(self.name+" ",prep,"-", prep['reagent_label'])
-                raise TypeError
+                logging.debug("Missing field for making the sample run id :"+self.name+" ",prep,"-", prep['reagent_label'])
```

Comment:
There's still so much things in that code that don't make sense.

On Wed, Apr 29, 2015 at 4:32 PM, Guillermo Carrasco <
notifications@github.com> wrote:

> In LIMS2DB/objectsDB/objectsDB.py
> <https://github.com/SciLifeLab/LIMS2DB/pull/12#discussion_r29341023>:
>
> > @@ -459,8 +459,7 @@ def _make_sample_run_id(self, seq_art, lims_run, prep, run_type):
> >                  samp_run_met_id = '_'.join([lane, date, fcid, barcode])
> >              except TypeError:
> >                  #happens if the History object is missing fields, barcode might be None
> > -                logging.debug(self.name+" ",prep,"-", prep['reagent_label'])
> > -                raise TypeError
> > +                logging.debug("Missing field for making the sample run id :"+self.name+" ",prep,"-", prep['reagent_label'])
>
> dunno man, but you fixed it [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/LIMS2DB/pull/12/files#r29341023>.
>